app/controllers/resources_controller.rb

```diff
@@ -70,6 +70,16 @@ def show
 
   private
 
+  CREATE_PARAMS_EXCLUDE_FROM_COCINA = [:action, :controller, :resource, :accession, :priority, :assign_doi]
+
+  def cocina_create_params
+    params.except(*CREATE_PARAMS_EXCLUDE_FROM_COCINA).to_unsafe_h
+  end
+
+  def cocina_update_params
+    params.except(:action, :controller, :resource, :id).to_unsafe_h
```

Comment:
It wasn't clear to me why creates need to ignore more than updates. Is that worth having a note about?

Reply:
I don't know why. This is just how the code was written. I just moved it to separate method which makes it easier to understand.